stock_picking_update_responsible/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Stock Picking Update Responsible",
+    "summary": """
+        Update responsible when user print picking operations.
+        Now only sales administrators and picking preparer can modify the responsible.
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "17.0.1.0.0",
+    "category": "Stock",
+    "website": "https://github.com/solvosci/slv-stock",
+    "depends": [
+        "stock",
```

Comment:
Then, new dependency should be this one:
```suggestion
        "sale_stock",
```